gpkit/tests/t_sub.py

```diff
@@ -203,115 +207,6 @@ def setup(self):
         sol = Top().solve(verbosity=0)
         self.assertAlmostEqual(sol['cost'], 2)
 
-        # add unit test for varsbyname, vector composition
-        # @whoburg: and some for unions and intersections before solving
-
-    # def test_simpleaircraft(self):
```

Comment:
It's a bit out of date; do you think the tests make sense as things we want to accomplish?